include/espeak.md

```diff
@@ -55,3 +58,24 @@ any changes in our `sconscript` file.
 Due to problems with emoji support (causing crashes), emoji dictionary files are being excluded
 from the build, they are deleted prior to compiling the dictionaries in the
 `<nvda repo root>/nvdaHelper/espeak/sconscript` file.
+
+## Manually testing Espeak-ng
+If you wish to test Espeak-ng directly, perhaps to create steps to reproduce when raising an issue for the espeak-ng project.
+Consider feeding SSML directly to the executable provided by the project:

```

Comment:
```suggestion
If you wish to test eSpeak-ng directly, perhaps to create steps to reproduce when raising an issue for the eSpeak-ng project, consider feeding SSML directly to the executable provided by the project.
```

Reply:
Yep, that is what I intended to say. Good catch.